include/libhal/percentage.hpp

```diff
@@ -21,6 +23,28 @@ class percentage
   constexpr static float_t min = -1.0;
   constexpr static float_t zero = 0.0;
 
+  constexpr static bool nearly_equal(percentage p_percentage1,
```

Comment:
```suggestion
  constexpr static bool equals(percentage p_percentage1,
```

Change the name to `equals`. I would like to overload this against other types as well and have the name be consistent across them.